src/gpu_marker.jl

```diff
@@ -1,5 +1,12 @@
 module GPUMarker
     import ..LIKWID: liblikwid
+    using Libdl
+
+    function issupported()
+        dlopen(liblikwid) do handle
```

Comment:
Maybe cache this result?

Reply:
Sure. I used a `Ref{Union{Nothing, Bool}}` for this. Is there a better way?

(Another option could be to check for gpu support in init and `@eval GPUMarker issupported() = true / false`.)